tests/h/views/activity_test.py

```diff
@@ -113,7 +116,8 @@ class TestGroupSearchController(object):
 
     """Tests unique to GroupSearchController."""
 
-    def test_renders_join_template_when_no_read_permission(self, controller, pyramid_request, group):
+    @pytest.mark.parametrize('test_group', GROUP_TYPE_OPTIONS, indirect=['test_group'])
```

Comment:
Ah, interesting. I hadn't come across the `indirect` facility in pytest before. From a quick skim through existing uses of `pytest.mark.parametrize` it looks like we just haven't needed to use fixtures involving DB objects in parametrized tests before?